src/pages/index.tsx

```diff
@@ -1,19 +1,126 @@
-import { Flex, Text } from "@chakra-ui/react";
+import {
+  Flex,
+  Text,
+  Box,
+  Heading,
+  Button,
+  HStack,
+  Divider,
+  Stack,
+  Link,
+} from "@chakra-ui/react";
 import Image from "next/image";
 import FirstLandingImage from "public/images/landing_first.svg";
 import SecondLandingImage from "public/images/landing_second.svg";
 import ThirdLandingImage from "public/images/landing_third.svg";
 
 function LandingPage() {
   return (
-    <Flex height="100%" justifyContent="space-around" alignItems="center">
-      <Text fontSize="2xl">Landing Page.</Text>
-      <Image
-        src={FirstLandingImage}
-        alt="Build Websites"
-        width={500}
-        height={500}
-      />
+    <Flex direction="column" h="calc(100vh - 100px)">
+      <Stack
+        justifyContent="center"
+        alignItems="center"
+        direction={{ base: "column-reverse", md: "row" }}
+        marginBottom="10"
+        flex="1"
+        flexBasis="250px"
+      >
+        <Box bg="white" w={{ md: "50%" }} p={10}>
+          <Heading
+            fontSize={{ base: "x-large", md: "xx-large", lg: "xxx-large" }}
+            marginBottom="1"
+          >
+            Non-profit Portal
+          </Heading>
+          <Text fontSize="medium" marginY="5" maxW={500}>
+            We build software, websites, and mobile apps to turn your ideas into
+            into real products. If you have an existing project or a happy to
+            get involved in your development cycle.
+          </Text>
+          <Button colorScheme="blue">Log In</Button>
+        </Box>
+        <Image
+          src={FirstLandingImage}
+          alt="Build Websites"
+          width={250}
+          height={250}
+        />
+      </Stack>
+      <Stack
+        justifyContent="center"
+        alignItems="center"
+        direction={{ base: "column", md: "row" }}
+        marginBottom="10"
+        flex="1"
+      >
+        <Image
+          src={SecondLandingImage}
+          alt="Build Websites"
+          width={190}
+          height={190}
+        />
+        <Box w={{ md: "50%" }} p={10} marginLeft={{ md: "20" }}>
```

Comment:
Apply a max width to this box so that the description is more condensed and the entire stack is centered.  You will need to do the same to the stack below.  This is a good example of why a separate component is useful, since you only need to make one change to fix both.